### ISSUE TITLE
many packages needs updated PKGBUILDS.

### ISSUE THREAD

**Issue by Strit** [manjaro-arm/packages-extra#4]
artwork-maia-gnome = missing source
~~manjaro-settings-manager = missing mhwd files (dependency issue)~~
~~ttf-aboriginal-sans = missing package() section~~
~~ttf-thaana-fonts = missing package() section~~

~~The packages artwork-maia and breeze-maia-icon-themes has this error:~~

~~> CMake Error at CMakeLists.txt:8 (find_package):~~
~~>  Could not find a package configuration file provided by "ECM" (requested~~
~~>  version 1.5.0) with any of the following names:~~
~~>~~
~~>    ECMConfig.cmake~~
~~>    ecm-config.cmake~~
~~>~~
~~> Add the installation prefix of "ECM" to CMAKE_PREFIX_PATH or set "ECM_DIR"~~
~~>  to a directory containing one of the above files.  If "ECM" provides a~~
~~>  separate development package or SDK, be sure it has been installed.~~

I will add more if/when I find more.

**Comment by Strit** [comment]
Think I fixed MSM. Added mhwd. Removed mhwd's nvidia and catalist/ati dependencies. Also removed v86d dependency from mhwd. So that builds fine now. v86d is an AUR package, if we ever need it back.

**Comment by dodgejcr** [comment]
Yeah, github went down on my last night as I was trying to finish all these updates. artwork-maia-gnome had died on upstream and mainly replaced by the gtk theme. I will remove it from the repo.  

Those were the fixes I needed to get mhwd working last time so that seems correct.

I assume by the markout, you got the fonts packages working?

and I will look into the other packages today. 
EDIT: I only have an x86_64 machine on me right now and tried to build artwork-maia and breeze. Everything worked fine but I did have to update cmake. Im using version  3.6.1.
